README.md

```diff
@@ -1,3 +1 @@
-# Confluent Commons + Docker
-
-Deprecated. Just use common!
+# Common Docker Utilities
```

Comment:
Can you add a longer description about this repo? At the minimal, link to the README in base & utility-belt?